frontend/src/store/reduxState.ts

```diff
@@ -0,0 +1,58 @@
+export interface IUser {
+  avatarUrl?: string;
+  channelId: number;
+  creationDate: string;
+  numFollowers: number;
+  partnered: boolean;
+  username: string;
+  emotes: any;
+}
+
+export interface ISystem {
+  editingEmote: boolean;
```

Comment:
This "editingEmote" feels like a local component state. I'm guessing it doesn't really belong in global app state?  We should consider moving this to the local state of the component that needs it.